Autoloads/BasicData.gd

```diff
@@ -275,6 +267,10 @@ func remove_disabled_methods(method_list : Array, exceptions : Array) -> void:
 			method_list.remove(index)
 
 func remove_thing(thing : Object) -> void:
+	if !thing:
+		return
+	if !is_instance_valid(thing):
+		return
```

Comment:
Why this two additional checks are needed here?
When this function will get invalid object, then it should crash(or broke script) so user will know that something strange happens.
I tested script without this lines and everything works fine

Reply:
Yeah perhaps not really needed.